pkg/gadget-collection/gadget-collection.go

```diff
@@ -39,7 +39,7 @@ import (
 	snisnoop "github.com/kinvolk/inspektor-gadget/pkg/gadget-collection/gadgets/trace/sni"
 	tcptracer "github.com/kinvolk/inspektor-gadget/pkg/gadget-collection/gadgets/trace/tcp"
 	tcpconnect "github.com/kinvolk/inspektor-gadget/pkg/gadget-collection/gadgets/trace/tcpconnect"
-	traceloop "github.com/kinvolk/inspektor-gadget/pkg/gadget-collection/gadgets/traceloop"
+	traceloop "github.com/kinvolk/inspektor-gadget/pkg/gadget-collection/gadgets/trace/loop"
```

Comment:
Could it be added to `local-gadget interactive` in TraceFactoriesForLocalGadget too? That would be easier to debug than to use Kubernetes.